retrieval-market.md

```diff
@@ -69,13 +69,13 @@ The query should include the CID of the piece that the client is interested in r
 
 ```sh
 type RetQuery struct {
-    ## TODO: what exactly does this link to?
+	## TODO: what exactly does this link to?
 	piece Link
 }
 
 type RetQueryResponse union {
```

Comment:
unsure what the intent here was, keyed unions need a key, but perhaps there was something else going on? my keys might be too verbose or there might be an alternative approach appropriate here